include/ideep/tensor.hpp

```diff
@@ -259,6 +263,30 @@ class tensor : public memory {
           strides[w] == 1;
     };
 
+    inline bool is_channels_last() const {
```

Comment:
Do you assume `caffe2_use` is true in this implementation?
https://github.com/Guobing-Chen/ideep/blob/pytorch_dnnl/include/ideep/tensor.hpp#L221

Reply:
Regarding is_channels_last(), as from the code history, it comes from IPEX/Ideep and only used in IPEX/Ideep Conv OP. Caffe2 does not use is_channels_last(), while does use is_nhwc()/is_nchw().

Reply:
@jgong5 , any more suggestion for this PR?